src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -103,14 +104,15 @@ export class EntityCountDashboardComponent
 
   goToChildrenList(filterId: string) {
     const params = {};
-    params[this.groupBy] = filterId;
+    params[this.groupBy[0]] = filterId;
 
-    this.router.navigate([this._entity.route], { queryParams: params });
+    this.router.navigate([Entity.route], { queryParams: params });
```

Comment:
did you test the navigation in the app (clicking on one of the lines in the widget (e.g. the "female" children))? I think `Entity.route` will not work.

Reply:
Yeah! true. I have resolved it